Sources/Markdown/Inline Nodes/Inline Containers/Link.swift

```diff
@@ -63,6 +63,16 @@ public extension Link {
         }
     }
 
+    var isAutolink: Bool {
+        guard let destination,
+              childCount == 1,
+              let text = child(at: 0) as? Text,
+              destination == text.string else {
+            return false
+        }
+        return true
+    }
```

Comment:
I evidently don't know enough about Swift to understand how this works. The test you wrote sets a value to this property, but i was under the assumption that computed properties like this could not be written to. How does this allow for a setter?